styles/style.css

```diff
@@ -1,387 +1,305 @@
-/* universal styles */
 html {
   box-sizing: border-box;
   font-family: "Montserrat", sans-serif;
   font-size: 16px;
   color: rgb(255, 255, 255);
   background-color: black;
 }
+
 *,
 *:before,
 *:after {
   box-sizing: inherit;
   padding: 0;
   margin: 0;
 }
+
 img {
   display: block;
   max-width: 100%;
 }
 
-/* mobile */
-/* landing-section */
-.container-1-pc,
-.box-container,
-.container-2-pc {
-  display: none;
-}
-
-.landing-section {
-  height: 100vh;
-  width: 100%;
-  background-image: url(/images/balkouras-nicos--tJcdoXAtlQ-unsplash.jpg);
-  background-position: center;
-  background-size: cover;
-  display: flex;
-}
-
-.mobile {
-  margin: auto;
-  text-align: center;
-}
-
-.mobile h1 {
-  padding: 1rem;
+a {
+  text-decoration: none;
+  color: white;
 }
 
-#mobile {
+a:hover {
+  cursor: pointer;
   text-decoration: underline;
-  font-weight: 900;
-  font-size: 2.3rem;
+  color: red;
 }
-/* tablet */
-@media screen and (min-width: 768px) {
-  .landing-section {
-    flex-direction: column;
-    justify-content: space-between;
-  }
-  .mobile {
-    display: none;
-  }
-  .container-1-pc,
-  .box-container,
-  .container-2-pc {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 2rem;
-  }
 
-  .landing-section h1 {
-    font-size: 2.5rem;
-  }
+:root {
+  --dvh-with-fallback: 100vh;
+  --padding: 2rem;
+  --padding-big: 5rem;
```

Comment:
Checkout [clamp](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp )
Defining variables with clamp can really help you with responsiveness and making the site look fine on all display sizes

calc is also a good option, for example if you have a variable which you like but you need it 20% bigger you can do:

width: calc( var(--varname) * 1.2);